content/docs/visualization/template_reference.md

```diff
@@ -54,6 +54,7 @@ If functions are used in a pipeline, the pipeline value is passed as the last ar
 | humanize      | number        | string  | Converts a number to a more readable format, using [metric prefixes](http://en.wikipedia.org/wiki/Metric_prefix). 
 | humanize1024  | number        | string  | Like `humanize`, but uses 1024 as the base rather than 1000. |
 | humanizeDuration | number     | string  | Converts a duration in seconds to a more readable format. |
+| humanizeTimestamp | number    | string  | Converts a Unix timestamp in seconds to a human-readable [RFC3339](https://www.ietf.org/rfc/rfc3339.txt)-style time string. |
```

Comment:
The paragraph below allows us to improve output the humanized formats in the future. If we want to offer an explicit RFC3339 view, then a rfc3339Timestamp function would be more future proof.